pkgs/top-level/perl-packages.nix

```diff
@@ -10029,12 +10031,13 @@ let
       url = "mirror://cpan/authors/id/K/KU/KURIHARA/Imager-QRCode-0.035.tar.gz";
       sha256 = "2a848deba29eb3942c44709a6853e318acab0c468cbfedbb9baae54760032513";
     };
-    propagatedBuildInputs = [ Imager ];
```

Comment:
this may be wrong, it is one of the packages you recently added, and cpan2nix has no glue. it took deps from cpan metadata, which is often invalid